scripts/build.sh

```diff
@@ -1,17 +1,23 @@
 #!/usr/bin/env bash
 
-if [ -z "$1" ]; then 
+if [ -z "$1" ]; then
     BUILD_TYPE=Release
-else 
+else
     BUILD_TYPE="$1"
 fi
 
-cd $(dirname "$0")
-mkdir -p build
-cd build
-cmake .. -DCMAKE_BUILD_TYPE="$BUILD_TYPE"
+cd $(dirname "$0")/.. &&
```

Comment:
It may not be present. Every other build/release script is written with the assumption it cannot require the `.git` directory (and subsequently the use of the `git` command).